lib/chitter.rb

```diff
@@ -0,0 +1,94 @@
+require 'sinatra/base'
+require 'data_mapper'
+require 'rack-flash'
+
+# ----- DATAMAPPER SETUP BEGIN --------#
+
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, "postgres://localhost/chitter_#{env}")
+
+require './lib/user'
+require './lib/peep'
+
+DataMapper.finalize
+
+DataMapper.auto_upgrade!
+
+# ------ DATAMAPPER SETUP END ---------#
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  set :session_secret, 'seekrit' # why does this have to be set?
+
+  use Rack::Flash
+  use Rack::MethodOverride
+
+  get '/' do
+    @peeps = Peep.all
+    @peeps.reverse!
```

Comment:
Will this guarantee reverse chronological order?  Can this logic be pushed down into the data layer using DataMapper?